dascore/examples.py

```diff
@@ -451,6 +451,24 @@ def random_spool(time_gap=0, length=3, time_min=np.datetime64("2020-01-03"), **k
     return dc.spool(out)
 
 
+@register_func(EXAMPLE_SPOOLS, key="radom_directory_das")
```

Comment:
*random

Reply:
Also, currently getting the following error trying to getting the spool:
```python
import dascore as dc
sp  = dc.get_example_spool("radom_directory_das")

AttributeError: module 'dascore.io.segy.core' has no attribute 'SegyV2'. Did you mean: 'SegyV2_0'?
```
I see you've already merged master so I'm unsure what the problem is.